views/scss/qti/_associate.scss

```diff
@@ -18,7 +18,6 @@
         & > .qti-choice {
             min-width: 100px;
             min-height: 30px;
-            max-width: 180px;
```

Comment:
Quick fix :)
But I wonder if it is not too aggressive, see my previous comment.

BTW, this line has not changed for 7y, the last change is from https://github.com/oat-sa/extension-tao-itemqti/commit/978c2f165e83e8f2e285df879a1c453402719234#diff-f2f402bcd0bcb674fae83ec4cd965a7c21527cb011203c050271d161c2a42ab6L21